TypeScript.YAML-tmLanguage

```diff
@@ -26,11 +26,18 @@ repository:
       '1': { name: variable.ts }
     end: (?=$|[;,])
     patterns:
+    - include: '#ternary-operator'
     - include: '#type-annotation'
     - include: '#string'
     - include: '#comment'
     - include: '#expression'
 
+  ternary-operator:
```

Comment:
Agreed, it is not safe to end here since even in other scenarios like the following, highlighting shows no issues, though it should. 
```typescript
var x = true ? {a: 42} : {b: 43};
var y = true ? {a: 42} : let var2 = 8;
```
I will add a bigger scope called ternary-expression that only allows this ternary-operator followed by an expression, and see if that works. Thanks.